### ISSUE TITLE
request for BatchNormalization3D

### ISSUE THREAD

**Issue by sharpstill** [chainer/chainer#4753]
Is chainer.links.BatchNormalization can only support 2D feature map ? How to write BatchNormalization3D in chainer? 
Furthermore, pytorch has already support the BN3D here:
https://pytorch.org/docs/master/nn.html?highlight=batchnorm3d#torch.nn.BatchNorm3d
it is 5D input (a mini-batch of 3D inputs with additional channel dimension)

**Comment by delta2323** [comment]
Hi, `L.BatchNormalization` supports `axis` option, which specifies axes along which we take averages. I guess BN can accept images other than 2D by using that.

**Comment by sharpstill** [comment]
@delta2323 no, `L.BatchNormalization` does not have `axis `option.  following
https://docs.chainer.org/en/stable/reference/generated/chainer.links.BatchNormalization.html?highlight=batch%20normalization

**Comment by disktnk** [comment]
`axis` option merged at #4671 will be released on next v5.0.0b1

**Comment by delta2323** [comment]
Ah, sorry. Yes, as @disktnk pointed out, it is to be introduced in the next major version.

**Comment by kmaehashi** [comment]
I've replied to your question on SO.
https://stackoverflow.com/questions/50303841/how-to-use-batchnormalization3d-in-chainer
